### ISSUE TITLE
Microsoft.data.sqlclient Support for EF6

### ISSUE THREAD

**Issue by Hassan3152** [dotnet/SqlClient#1450]
Is there any plan to support EF 6 in Microsoft.data.sqlclient as i can't authenticate with azure sql using authentication type "Active directory managed identity" with EF 6.

**Comment by ErikEJ** [comment]
Yes, possible today: https://www.nuget.org/packages/ErikEJ.EntityFramework.SqlServer/

**Comment by roji** [comment]
Making Entity Framework 6 (non-core) work with Microsoft.Data.SqlClient is part of the EF team plan for the coming year (see https://docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-7.0/plan#theme-clear-path-forward-from-ef6).
